docs/lakeFSBountyHunterProgram.md

```diff
@@ -0,0 +1,96 @@
+lakeFS Bounty Hunter Program
```

Comment:
this is missing a header that connects all our new pages together. 
@hiccups28 take a look at this one as an example - https://raw.githubusercontent.com/treeverse/lakeFS/master/docs/contributing.md